### ISSUE TITLE
travis setup openshift fails

### ISSUE THREAD

**Issue by dreyks** [travis-ci/travis.rb#346]
I'm running `travis setup openshift` in my project dir and getting an error `Invalid scheme format: git@github.com`. The error is from `Addressable` gem 

`git remote -v` executed in project folder yields `origin  git@github.com:<username>/<reponame>.git` but `Addressable` seems not to like this kind of notation
```console
$ git remote -v
origin  git@github.com:<username>/<reponame>.git (fetch)
origin  git@github.com:<username>/<reponame>.git (push)
```
```console
$ irb
2.2.3 :001 > require 'addressable'
 => true
2.2.3 :002 > Addressable::URI.parse('git@github.com:travis-ci/travis.rb')
Addressable::URI::InvalidURIError: Invalid scheme format: git@github.com
```

Am I doing something wrong?

**Comment by shioyama** [comment]
Seeing the same issue here.

**Comment by BanzaiMan** [comment]
I think this is a duplicate of #342, which should be fixed by 1.8.1, which I plan to release tomorrow.
